jupyterhub/jupyterhub/overlays/jupyterhub-idle-culler/deploymentconfig.yaml

```diff
@@ -20,9 +20,14 @@ spec:
                 secretKeyRef:
                   name: jupyterhub-idle-culler
                   key: token
+            - name: CULLER_TIMEOUT
+              valueFrom:
+                configMapKeyRef:
+                  name: jupyterhub-cfg
+                  key: culler_timeout
           command:
             - jupyterhub-idle-culler
-            - --timeout=259200  # 3 days
+            - --timeout=${CULLER_TIMEOUT}  # in seconds
```

Comment:
The env var will need to use parenthesis `$(CULLER_TIMEOUT)` or the pod will fail to run

```python
Traceback (most recent call last):
File "/opt/app-root/bin/jupyterhub-idle-culler", line 8, in <module>
sys.exit(main())
File "/opt/app-root/lib/python3.6/site-packages/jupyterhub_idle_culler/__init__.py", line 409, in main
parse_command_line()
File "/opt/app-root/lib/python3.6/site-packages/tornado/options.py", line 707, in parse_command_line
return options.parse_command_line(args, final=final)
File "/opt/app-root/lib/python3.6/site-packages/tornado/options.py", line 351, in parse_command_line
option.parse(value)
File "/opt/app-root/lib/python3.6/site-packages/tornado/options.py", line 575, in parse
self._value = _parse(value)
ValueError: invalid literal for int() with base 10: '${CULLER_TIMEOUT}'
```